packages/react/src/legend/LocationTotalsLegend.tsx

```diff
@@ -22,6 +22,9 @@ import Disc from './Disc';
 export interface Props {
   diff?: boolean;
   colors?: Colors | DiffColors;
+  unequalText?: string;
```

Comment:
The naming is confusing here. Let's make one object prop instead named `textMessages`  with child props `aboutEqual`, `moreOutgoing`, and `moreIncoming`

Reply:
Or leave them as separate props, just change the names, as they are wrong.